helm/prometheus-rules/templates/recording-rules/cortex.rules.yml

```diff
@@ -222,3 +222,6 @@ spec:
     # Falco event counts
     - expr: sum(falco_events{priority=~"0|1|2|3|4|5|6|7"}) by (cluster_type, cluster_id, priority, rule)
       record: aggregation:falco_events
+    # Starboard unique vulnerability counts by severity
+    - expr: count(count by (vulnerability_id, severity) (starboard_exporter_vulnerabilityreport_image_vulnerability)) by (severity)
```

Comment:
do you wanna add the cluster id?

Reply:
Yep, I probably do. Dang it. Thanks Joe, another PR incoming

Reply:
well apparently i do not remember how this works. bam!